### ISSUE TITLE
Keras Gesture Classifier Training

### ISSUE THREAD

**Issue by denver2714** [denver2714/Hand-Gesture-Control-Keras-Project#12]
Build and train a Keras neural network to classify hand gestures from landmark data.

Build a Keras Sequential model:
Input(63) → Dense(128, relu) → Dropout(0.3) → Dense(64, relu) → Dropout(0.3) → Dense(5, softmax)
